flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -687,7 +687,7 @@ else if (current == RUNNING || current == DEPLOYING) {
 			}
 
 			else if (current == FINISHED || current == FAILED) {
-				// nothing to do any more. finished failed before it could be cancelled.
+				// nothing to do any more. finished/failed before it could be cancelled.
```

Comment:
unrelated change

Reply:
thats why it is in a separate hotfix commit